### DISCUSSION TITLE
Possible to not serialize null?

### DISCUSSION THREAD

**Discussion by felixgabler** [flyerhq/flutter_chat_ui#210] [Ideas & Requests]
Since the models use `json_serializable`, I was wondering whether we could make it possible to disable the behavior where null values are included in the JSON.

https://pub.dev/documentation/json_annotation/latest/json_annotation/JsonKey/includeIfNull.html

This would keep the database smaller since there are often quite a few null values (e.g. most of the author)

**Comment by demchenkoalex** [comment]
Hmm interesting suggestion, I will read more about that. Thanks!

**Reply by felixgabler** [reply]
This would be the changes necessary: https://github.com/flyerhq/flutter_chat_types/pull/18

**Reply by demchenkoalex** [reply]
Thanks. Merged :)
